packages/jmix-react-antd/src/util/history.ts

```diff
@@ -1,7 +1,7 @@
-import {addPagingParams} from "../ui/paging/Paging";
-import {JmixPagination} from "@haulmont/jmix-react-core";
+// import {addPagingParams} from "../ui/paging/Paging";
+// import {JmixPagination} from "@haulmont/jmix-react-core";
 
-export function saveHistory(routingPath: string, pagination?: JmixPagination) {
-  const {current, pageSize} = pagination ?? {};
-  window.history.pushState({}, '', addPagingParams(routingPath, current, pageSize));
+export function saveHistory(/* routingPath: string, pagination?: JmixPagination */) {
+  // const {current, pageSize} = pagination ?? {};
+  // window.history.pushState({}, '', addPagingParams(routingPath, current, pageSize));
```

Comment:
We should not have commented-out code unless we have a very good reason.

Reply:
It doesn't make much sense to store these params until we discover a proper way to reflect screen api state in the url

Reply:
Let's delete the code. We can always restore it using git.